Makefile

```diff
@@ -453,6 +453,7 @@ cervantesupdate: all
 	# Cervantes launching scripts
 	cp $(COMMON_DIR)/spinning_zsync $(INSTALL_DIR)/koreader/spinning_zsync.sh
 	cp $(CERVANTES_DIR)/*.sh $(INSTALL_DIR)/koreader
+	cp $(CERVANTES_DIR)/spinning_zsync $(INSTALL_DIR)/koreader
```

Comment:
What happens with the readme and those service files? Is it supposed to be specific things or basically just `/*`?

Reply:
That's probably why the first one is `*.sh` ;).

The second one is a single file, so it shouldn't pull anything else.

And if you were asking where the other stuff goes: I don't know ^^.